src/app.md

```diff
@@ -725,6 +726,56 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given a list of version `&str`s, find the latest [`semver::Version`] that satisfies the given range
+"<= 1.0.0". A [`semver::VersionReq`] is used to compare the parsed version to a 
+minimum version requirement.  Filter for version `&str` that are verified as [`semver::Version`] with [`Version::parse`] and satisfies the [`semver::VersionReq`] using [`semver::VersionReq::matches`](https://docs.rs/semver/*/semver/struct.VersionReq.html#method.matches).  Return the latest [`semver::Version`] using [`std::cmp::max`](https://doc.rust-lang.org/std/cmp/fn.max.html).
```

Comment:
To ease readability, links are placed at the bottom of the file in alphabetic order.

No need for double space after a dot. Maybe I am too picky? :-)
- "requirement.  Filter"
- "matches).  Return"